RELEASE.md

```diff
@@ -1,3 +1,48 @@
+# Release 1.12.0
+
+## Major Features and Improvements
+* tf.data optimizations. Users can now represent, get and set options of a tf.data pipeline.
+  tf.data.AUTOTUNE enables the level of parallelism to be determined at runtime.
+* Keras models can now be directly exported to the SavedModel format and used with Tensorflow Serving.
+
+## Breaking Changes
+
+## Bug Fixes and Other Changes
+
+* tf.data:
+  * All C++ code moves to the `tensorflow::data` namespace.
+  * Support evaluating with a tf.data.Dataset
+  * Adding support for `num_parallel_calls` to `tf.data.Dataset.interleave`.
+* `tf.contrib`:
+  * Remove tf.contrib.linalg.  Library now in tf.linalg.
+  * Replace any calls to tf.contrib.get_signature_def_by_key(metagraph_def, signature_def_key) with meta_graph_def.signature_def[signature_def_key]. Catching a ValueError exception thrown by tf.contrib.get_signature_def_by_key should be replaced by catching a KeyError exception.
+  * Keras model can be exported to the SavedModel format using tf.contrib.saved_model.save_keras_model().
```

Comment:
Sounds similar to the note in "Major Features":
"Keras models can now be directly exported to the SavedModel format and used with Tensorflow Serving."

Can you merge these two? May be put:
"Keras models can now be directly exported to the SavedModel format (tf.contrib.saved_model.save_keras_model()) and used with Tensorflow Serving. "
in Major Features?

Reply:
Done.